shared/datastore.go

```diff
@@ -21,23 +21,20 @@ func LoadTestRun(ctx context.Context, id int64) (*TestRun, error) {
 	return &testRun, nil
 }
 
-// LoadTestRuns loads the TestRun entities for the given parameters.
+// LoadTestRunKeys loads the keys for the TestRun entities for the given parameters.
```

Comment:
Can we get tests for the two new functions in this file?

Reply:
Related note: GitHub's crappy diff is making it hard to do a thorough review of this file. I'm sure once I review your tests for the functions I'll be confident that the code is correct :smiley: